others/career-development.md

```diff
@@ -0,0 +1,73 @@
+# Career Development
+
+## CV
+
+### Best practice
+
+- Keep it short and concise, 1-2 pages maximum
+- Put your details at the top. Only these details are necessary:
+  - Name
+  - Email
+  - Phone number
+  - Your city (not your address since this is personal information! If they need it they'll ask for it later)
+- Use the following sections, and only populate them with relevant details starting from the most recent first
+  1. Education
+  2. Employment
+     - Use action verbs for each bullet point (see: <https://www.indeed.com/career-advice/resumes-cover-letters/action-verbs-to-make-your-resume-stand-out>)
+     - Show your accomplishments, with numbers if you can (e.g. Improved landing page time to interactive by 50%)
+  3. Skills
+  4. Personal projects
+  5. Personal interests (optional)
+
+### Resources
+
+Guide to writing a CV
+
+- <https://www.freecodecamp.org/news/writing-a-killer-software-engineering-resume-b11c91ef699d/>
+
+CV sample
+
+- <https://careercup.com/resume>
+
+## Interviews
+
+Interview questions with answers
+
+- <https://github.com/FAQGURU/FAQGURU>
```

Comment:
Hmm, some of these are going to be a bit difficult for our students as there's a lot of computer science stuff in there e.g. https://github.com/FAQGURU/FAQGURU/blob/master/topics/en/javascript.md#being-told-that-an-unsorted-array-contains-n---1-of-n-consecutive-numbers-where-the-bounds-are-defined-find-the-missing-number-in-on-time. The concept is solid though, maybe we could select some interview questions they wouldn't find too hard to practice with?

Reply:
Yeah, my idea for this Career Development page is that it can also serve as a reference for graduates who have been working for a while. I'll add a note that these interview questions are more intermediate-level. If I find any beginner-level resources I'll add them.